baby-liminal-extension/src/backend/tests.rs

```diff
@@ -36,11 +36,6 @@ fn simulate_verify<Exc: BackendExecutor>(expected_ret_val: u32) {
     assert_eq!(charged, expected_charged);
 }
 
-#[test]
-fn extension_is_enabled() {
-    assert!(BabyLiminalChainExtension::<AlephRuntime>::enabled())
-}
-
```

Comment:
why not negate the assertion?

Reply:
because soon it will be by default enabled again... but okay, reverted change to show current state

Reply:
oh, sorry, this is why: https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/7902869902/job/21569473967?pr=1616 

I forgot